src/Symfony/Bridge/Doctrine/PropertyInfo/DoctrineExtractor.php

```diff
@@ -200,6 +200,7 @@ public function getTypes($class, $property, array $context = [])
                             return [new Type(Type::BUILTIN_TYPE_ARRAY, $nullable, null, true, new Type(Type::BUILTIN_TYPE_INT), $enumType ?? new Type(Type::BUILTIN_TYPE_STRING))];
                     }
                 case Type::BUILTIN_TYPE_INT:
+                    // no break, because type int and string are handled the same way, if $enumType is set
```

Comment:
I meant BEFORE this case, aka between line 201 and 202 :)

Reply:
Oh yeah, kinda obvious, sorry for the misunderstanding :)

I've changed the commit and added the missing break.